[codex] Fix Feature Flags documentation guidance (#36573)
DataDog/documentation: content/en/feature_flags/server/python.md

## Overview

> This page describes how to instrument your Python application with the Datadog Feature Flags SDK. The Python SDK integrates with [OpenFeature][1], an open standard for feature flag management, and receives flag updates through Remote Configuration in the Datadog Python tracer (`ddtrace`).

Author: 🤖 **Context from Codex:** Clarifies that Python server-side flags receive Remote Configuration through the Datadog Python tracer (`ddtrace`).

This guide explains how to install and enable the SDK, create an OpenFeature client, and evaluate feature flags in your application.

## Prerequisites

Before setting up the Python Feature Flags SDK, ensure you have:

- **Datadog Agent** version 7.55 or later with [Remote Configuration][2] enabled
- **Datadog [API key][3]** configured on the Agent
- **Datadog Python SDK** `ddtrace` version 3.19.0 or later

> - **OpenFeature Python SDK** `openfeature-sdk` version 0.7.0 or later if you use provider event handlers to wait for initialization. If you do not use that pattern, version 0.5.0 or later is sufficient.

Reviewer: Conditional version requirement is unusual to express mid-bullet. Restructure for clarity:

```suggestion
- **OpenFeature Python SDK** `openfeature-sdk`: version 0.5.0 or later (version 0.7.0 or later required if you use provider event handlers to wait for initialization)
```

Also: the existing callout at line 222 already explains the 0.7.0-vs-0.5.0 distinction, so this prerequisite restatement may be redundant. Consider whether the bullet can just say "version 0.5.0 or later" and leave the conditional detail to the callout.

Before setting up the Python Feature Flags SDK, ensure you have:

- **Datadog Agent** version 7.55 or later with [Remote Configuration][2] enabled
- **Datadog [API key][3]** configured on the Agent
- **Datadog Python SDK** `ddtrace` version 3.19.0 or later

> - **OpenFeature Python SDK** `openfeature-sdk` version 0.7.0 or later if you use provider event handlers to wait for initialization. If you do not use that pattern, version 0.5.0 or later is sufficient.

Author: 🤖 **Context from Codex:** The Python page already warned that provider event handlers for waiting require OpenFeature SDK 0.7.0 or later, while prerequisites said 0.5.0. This reconciles the two statements without forcing 0.7.0 on users who do not use that waiting pattern.
